snappi_ixnetwork/ixnetworkapi.py

```diff
@@ -285,6 +285,29 @@ def add_error(self, error):
         else:
             self._errors.append(error)
 
+    def parse_location_info(self, location):
+        """It will return (chassis,card,port)
+        set card as 0 where that is not applicable"""
+        if re.search("/|;", location) is None:
+            raise Exception("Please specify location with supported connection status")
+        LocationInfo = namedtuple("LocationInfo", ["chassis_info",
+                                                   "card_info",
+                                                   "port_info"])
+        if ';' in location:
+            try:
+                (chassis_info, card_info, port_info) = location.split(';')
+            except Exception:
+                raise Exception("Please specify physical <chassis>;<card>;<port>")
```

Comment:
remove word "physical" as it refers to physical hardware, which would miss lead for vm